k8s/node-local-ha.yaml

```diff
@@ -232,26 +238,18 @@ spec:
           requests:
             cpu: 25m
             memory: 5Mi
-        args:
-        - -conf
-        - /etc/coredns/Corefile
+        args: [ "-conf", "/etc/coredns/Corefile" ]
         securityContext:
           privileged: true
         ports:
-        - containerPort: 53
-          name: dns
-          protocol: UDP
-        - containerPort: 53
-          name: dns-tcp
-          protocol: TCP
-        - containerPort: 9253
+        - containerPort: 9254
```

Comment:
also here is this the correct port?

Reply:
yes, because https://github.com/contentful-labs/coredns-nodecache/pull/22/files#diff-834c7a2e8a45e23862b3157dda324304R171